dev/devicelab/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   args: 1.6.0
   file: 6.0.0-nullsafety.2
+  http: 0.12.2
```

Comment:
Had this been missing before?

Reply:
Yeah, I had to move this manually from `dev_dependencies` below to here before running `flutter update-packages --force-upgrade`. I believe the removal of package:http from package:test 1.16.0-nullsafety.6 causes the command to remove it from `dev_dependencies` below, and it actually should be a dependency.